debian/rules

```diff
@@ -66,7 +66,8 @@ override_dh_auto_install:
 override_dh_epics_postinstall:
 	# no op
 
-override_dh_auto_test: # skip for development
+override_dh_auto_test:
+	LD_LIBRARY_PATH=$(shell pwd)/lib/$(EPICS_HOST_ARCH) make runtests
```

Comment:
@mdavidsaver, I realized that you had accidentally turned off the tests a long time ago so I decided to turn them back on. Do you have an idea how to do that in a cleaner way (without my dirty `LD_LIBRARY_PATH` hack)?